datadog/resource_datadog_dashboard.go

```diff
@@ -2335,6 +2335,10 @@ func getManageStatusDefinitionSchema() map[string]*schema.Schema {
 			Type:     schema.TypeString,
 			Required: true,
 		},
+		"summary_type": {
+			Type:     schema.TypeString,
+			Optional: true,
```

Comment:
Can we add description in this schema as well https://github.com/terraform-providers/terraform-provider-datadog/pull/396/files#diff-22ba5b0fbe5d1b3cdfe783aeb61bf51aR561

Reply:
Should we also set a validation here to ensure its one of the three types - https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#validatefunc or will these change?

Reply:
Thanks for the review. I added a description and validation. Please let me know if I should move the validation function somewhere or otherwise alter it (I don't use Go much at the moment).